### DISCUSSION TITLE
Simplified Chinese version of wiki

### DISCUSSION THREAD

**Discussion by Rad586** [constellation-mc/commander#1] [General]
https://github.com/Rad586/Commander-wiki-CN
Seems github doesn't support pull requests for wiki?

**Comment by Rad586** [comment]
Also, links like [this](https://github.com/constellation-mc/commander/blob/main/src/main/java/me/melontini/commander/builtin/events/EntityEvents.java) are broken. 

`https://github.com/constellation-mc/commander/blob/main/src/main/java/me/melontini/commander/builtin/events/EntityEvents.java`
👇
`https://github.com/constellation-mc/commander/blob/main/src/main/java/me/melontini/commander/impl/builtin/events/EntityEvents.java`

**Comment by zenfyrdev** [comment]
Oh, wow. Thank you very much. But I cannot merge this yet because GH wikis have terrible i18n support. I need to figure out how to support this.

Wiki PRs: Impossible. Wikis are either contributor-only or open to everyone. If I don't want the wiki to be vandalized, I can only merge/cherry-pick commits from other repos, and that's it.

I think a better option is to use vitepress/vuepress and host it separately (like Andromeda).

**Comment by zenfyrdev** [comment]
We did it, Reddit:

https://constellation-mc.github.io/commander/

https://github.com/constellation-mc/commander/tree/documentation

**Comment by Rad586** [comment]
Pull request created!
